src/Cli/dotnet/CommandFactory/CommandResolution/DepsJsonCommandResolver.cs

```diff
@@ -3,250 +3,248 @@
 
 using Microsoft.DotNet.Cli.Utils;
 using Microsoft.DotNet.Cli.Utils.Extensions;
-using Microsoft.DotNet.Tools.Common;
 using Microsoft.Extensions.DependencyModel;
 
-namespace Microsoft.DotNet.CommandFactory
+namespace Microsoft.DotNet.Cli.CommandFactory.CommandResolution;
+
+public class DepsJsonCommandResolver : ICommandResolver
 {
-    public class DepsJsonCommandResolver : ICommandResolver
-    {
-        private static readonly string[] s_extensionPreferenceOrder = new[]
```

Comment:
I wasnt aware we were still using this syntax anywhere. Good idea to fix it.

Reply:
I'll likely apply this more broadly in a follow-up PR.